src/dashboard/luasrc/controller/ids-priv/ids_tab.lua

```diff
@@ -1,49 +1,9 @@
 module("luci.controller.ids-priv.ids_tab", package.seeall)  --notice that new_tab is the name of the file new_tab.lua
     function index()
         entry({"admin", "ids_tab"}, firstchild(), "Alert Dashboard", 60).dependent=false  
```

Comment:
More of a request than a comment, could this be changed to "tinyHIPPO Dashboard"

Reply:
Done